RoslynSuppressions.txt

```diff
@@ -0,0 +1 @@
+0534
```

Comment:
Where exactly do you get this error? Maybe we can fix it.

Reply:
This only occurred on the one branch build and not the legacy build but the issue has been resolved. removing suppression